llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -3051,7 +3084,7 @@ static int show_main(int argc, const char *argv[]) {
         DebugInfoFilename.ArgStr + "' is provided");
 
   if (Filename == OutputFilename) {
-    errs() << sys::path::filename(argv[0])
+    errs() << sys::path::filename(argv[0]) << " " << argv[1]
            << ": Input file name cannot be the same as the output file name!\n";
```

Comment:
this is just an artifact that `show` options parses command lines differently with the other three subcommands (that are not updated).

When all subcommands are updated to be `cl::SubCommand`, or when we introduce the following code, this is going to look more formal.

```
// If we want to update one subcommand at a time
if  subcommand uses cl::SubCommand {
   // the new method in this PR
   handles argv list one way
}
else {
  // the existing method
  handles argv list the other way
}
```